tests/unit/transaction_id_test.py

```
@@ -0,0 +1,75 @@
+
+def test_from_string_invalid_format_raises_error():
+    """Test that invalid formats raise ValueError (covers the try-except block)."""
+    invalid_strings = [
+        "invalid_string",
+        "0.0.123.123456789",  # Missing @ separator
+        "0.0.123@12345",      # Missing . in timestamp
+        "0.0.123@abc.def",    # Non-numeric timestamp
+    ]
+    
+    for s in invalid_strings:
+        with pytest.raises(ValueError) as exc_info:
```

Ruff (0.14.10)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)